Dockerfile

```diff
@@ -62,9 +62,7 @@ ENV OPERATOR=instana-agent-operator \
     USER_UID=1001 \
     USER_NAME=instana-agent-operator
 
-RUN microdnf install yum \
```

Comment:
That command install additional packages which will increase the number of found CVEs